Library/Formula/gtimer.rb

```diff
@@ -0,0 +1,83 @@
+class Gtimer < Formula
+  desc "GTimer is a GTK application for timing how you spend your time among various projects and tasks with support for generating plain text and HTML reports."
```

Comment:
Please shorten the description. The formula’s name and description should fit under 80 characters.